terraform/terraform-modules/cognito-auth/lambda/requirements.txt

```diff
@@ -0,0 +1,4 @@
+requests==2.29.0
+urllib3<2
+python-jose
```

Comment:
Should there be a version number on `python-jose` in order to avoid the dependency confusion vulnerability?

Reply:
Added a version number.